HC.Tomek/HC.Tomek.Js.Ex3/containers.js

```diff
@@ -0,0 +1,122 @@
+                questionAnswerRadio[j][i].name = "answer";
+                questionAnswerRadio[j][i].onchange = onRadioClick(j,i);
+                questionAnswerText[j][i] = document.createElement("label");
+                questionAnswerText[j][i].innerHTML = questions[j].answears[i].text;
+                answerControl[j][i].appendChild(questionAnswerRadio[j][i]);
+                answerControl[j][i].appendChild(questionAnswerText[j][i]);
+                if (j>0) {
+                    answerControl[j][i].style.display = "none";
+                }
+                quizContainer.appendChild(answerControl[j][i]);
+
+            }
+
+        }
+        nextButton.addEventListener("click", function () {
+            
+            
+            if (currentQuestionNumber >= questionNumber-1) {
+                nextButton.style.display = "none";
+                questionStatus.style.display = "none";
+                questionText[currentQuestionNumber].style.display = "none";
+                for (var c = 0; c < questions[currentQuestionNumber].answears.length; c++) {
+                    answerControl[currentQuestionNumber][c].style.display = "none";
+                    if ((questionAnswerRadio[currentQuestionNumber][c].checked == true) &&
+                        (questions[currentQuestionNumber].answears[c].isCorrect == true)) {
+                        correctAnswers++;
+                    }
+                }
+                
+                resultPage(correctAnswers, questionNumber);
+            }
+            else {
+                questionText[currentQuestionNumber].style.display = "none";
+                for (var c = 0; c < questions[currentQuestionNumber].answears.length; c++) {
+                    answerControl[currentQuestionNumber][c].style.display = "none";
+                    if ((questionAnswerRadio[currentQuestionNumber][c].checked == true) &&
+                        (questions[currentQuestionNumber].answears[c].isCorrect == true)) {
+                        correctAnswers++;
+                    }
+                    }
+                
+                questionText[currentQuestionNumber + 1].style.display = "";
+                for (var g = 0; g < questions[currentQuestionNumber + 1].answears.length; g++) {
+                    answerControl[currentQuestionNumber + 1][g].style.display = "";
+                }
+                if (currentQuestionNumber >= questionNumber - 2) nextButton.innerHTML = "End";
+
+            }
+            currentQuestionNumber++;
+            questionStatus.innerHTML = (currentQuestionNumber + 1) + " / " + questionNumber + " ";
+            
+        });
+
+        });
+    
+}
+
+function resultPage(correctAnswers,questionNumber) {
+    var result = document.getElementById("result");
+    result.style.display = "";
+    var wynik = document.createElement("label");
+    wynik.innerHTML = "Wynik: " + correctAnswers +"/" + questionNumber;
+    result.appendChild(wynik);
+    
+}
+
+
+
```

Comment:
zwróć uwagę na styl - bardzo dużo zbitego kodu + dużo pustych linii. Mało czytelny kod przez to jest